MIDAS/src/sensor_data.h

```diff
@@ -78,10 +78,10 @@ struct GPS {
     float satellite_count = 0;
 };
 
-struct Magnetometer {
-    float mx = 0;
-    float my = 0;
-    float mz = 0;
+struct MagnetometerReading {
+    float mx;
```

Comment:
Why were the default values removed?

Reply:
Default values disallow struct initialization